init.lua

```diff
@@ -52,10 +54,15 @@ minetest.register_entity("gauges:hp_bar", {
 		local breath = player:get_breath() <= 10 and player:get_breath() or 11
 
 		if self.hp ~= hp or self.breath ~= breath then
+			local health_t = "health_"..hp..".png"
+			local breath_t = "breath_"..breath..".png"
+			if mt_53 and breath == 10 or breath == 11 then
+				breath_t = "blank.png"
+			end
```

Comment:
https://github.com/minetest/minetest/blob/master/builtin/game/constants.lua#L27
`(minetest.PLAYER_MAX_BREATH_DEFAULT or 11)` is the best check here.